src/scripts/utils/AyahRangeError.js

```diff
@@ -0,0 +1,13 @@
+class AyahRangeError extends Error{
```

Comment:
'class' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).